Makefile

```diff
@@ -103,7 +103,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+	OPERATOR_TEMPLATES=./templates KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
```

Comment:
setting the OPERATOR_TEMPLATE here is actually better than what we do in nova setting in programmatically in each test.  https://github.com/openstack-k8s-operators/nova-operator/blob/e6742035a36d7c865620511d25d5784feb721999/test/functional/nova_controller_test.go#L61